lib/cocoapods/generator/copy_resources_script.rb

```diff
@@ -71,6 +82,28 @@ def install_resources_function
         end
       end
 
+      # @return [String] The contents of the target xcassets script.
+      #
+      def target_xcassets
+
+        project_path = user_project_path.to_path
+        user_target_name = target_name.sub("Pods-", "")
+
+        script = <<-EOS 
+        require 'xcodeproj';
```

Comment:
I dont think this change is going to work -- cocoapods needs to generate a workspace that works even on machines that don't have cocoapods installed

Reply:
I see. Didn't take that into consideration. 

Could I check for that and use this method only on machines that have it installed, and fall back to the shell command otherwise?
Or would you prefer not to have different behaviors on different machines?